src/Compatibility/Core/src/Android/AppCompat/Platform.cs

```diff
@@ -35,6 +35,10 @@ internal class Platform : BindableObject, IPlatformLayout, INavigation, IDisposa
 				var view = bindable as VisualElement;
 				if (view != null)
 					view.IsPlatformEnabled = newvalue != null;
+
+				if (view.Handler == null && newvalue is IVisualElementRenderer ver)
+					view.Handler = new RendererToHandlerShim(ver);
```

Comment:
We want all views to have a Handler for consistency. This way if there's a shimmed control you can still access the native elements via the Handler. This will also make it easier to implement gestures because all of the native gesture code can just be changed to retrieve the native controls via the Handler